mutornadomon/config.py

```diff
@@ -24,6 +28,24 @@ def initialize_mutornadomon(tornado_app=None, publisher=None, publish_interval=N
     monitor = MuTornadoMon(external_interface, **monitor_config)
     monitor.start()
 
+    ioloop = IOLoop.current()
+    add_callback = ioloop.add_callback
+    utilization_stat = UtilizationCollector(monitor)
+
+    def measure_callback(callback):
+        @wraps(callback)
+        def timed_callback(*args, **kwargs):
+            with utilization_stat:
+                return callback(*args, **kwargs)
+
+        return timed_callback
+
+    @wraps(add_callback)
```

Comment:
Actually, the idea of the "collectors" is to have pluggable stats. The interface is to add a `start` and `stop` methods.
So can you move most of this code in a `start` method of the UtilizationCollector?
The `start` method will be passed the `monitor` instance, and you can access the ioloop as `monitor.io_loop` instead of using `IOLoop.current()`.
The context manager could also be a separate class in the file where `UtilizationCollector` is defined.